plugins/premium/tinymcespellchecker.md

```diff
@@ -189,6 +189,8 @@ tinymce.init({
 
 This option specifies the languages that can be set for content when working with multi-language content. The specified languages will be available from the [`language` toolbar drop-down menu button](#toolbarbuttons).  For a list of available languages, see: [Supported languages](#supportedlanguages).
 
+> **Important:** In {{site.productname}} 5.9 and later, `content_langs` has precedence over `spellchecker_select_languages`. In order for `spellchecker_select_languages` to work, do not use the `content_langs` option. For more information, see [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).
```

Comment:
```suggestion
> **Important:** In {{site.productname}} 5.9 and later, `content_langs` will override `spellchecker_select_languages`. To use the `spellchecker_select_languages` option, do not define the `content_langs` option. For information on the `content_langs` option and Spell Checker Pro, see: [Content appearance options - `content_langs`]({{site.baseurl}}/configure/content-appearance/#content_langs).
```
If the intention is to eventually deprecate the select_languages option, then we need to re-work this section, not just add a note.

Reply:
I believe we're deprecating this in 5.9 anyways? As this is literally just replacing the old spellchecker specific functionality.

Reply:
Changed this to just outright say the setting is deprecated in favour of the new one. There's also a note in the description of the new one saying we'll use this as a fallback, which should convey all the information we had available here.